### ISSUE TITLE
Hint text for radio/checkbox groups gets read out repeatedly by VoiceOver

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-frontend#5876]
## Description of the issue

When arrowing through a form with a radio (or checkbox) group that has a hint, VoiceOver on iOS reads out the hint text repeatedly.
While this is not a WCAG fail, it can be annoying to screen reader users.

## Steps to reproduce the issue

Open our [inline radios example](https://design-system.service.gov.uk/components/radios/inline/) and arrow through the page (VO + right arrow).

## Actual vs expected behaviour

I don't know what is supposed to happen according to the spec, I only know what happens in all other major screen readers. I don't know if this means it's a bug in VoiceOver or just a different way how VoiceOver does things.

VoiceOver reads out the hint text:

* when arrowing to the legend
* (obviously also when arrowing to the hint text itself)
* when arrowing to the first radio button
* when arrowing to the second (and potential subsequent) radio buttons

The second and third are to be expected, the first and last not.
All other major screen readers only read the hint text when arrowing to the first radio button they encounter (and to the hint text itself).

## Environment

This happens with VoiceOver on iOS but also to some degree on macOS.
(It's less bad on macOS as the hint text doesn't get read out when arrowing to the legend.)

Tested in Safari on an iPad with iOS version 18.3.1 and a MacBook with macOS version 15.3.2.
